src/main/java/com/stablekernel/rxespresso/RxIdlingResource.java

```diff
@@ -93,13 +98,12 @@ public void registerIdleTransitionCallback(ResourceCallback resourceCallback) {
     @Override
     public <T> Observable.OnSubscribe<T> onSubscribeStart(Observable<? extends T> observableInstance,
                                                           final Observable.OnSubscribe<T> onSubscribe) {
+
         int activeSubscriptionCount = subscriptions.incrementAndGet();
+        onSubscribeList.put(new WeakReference<Observable.OnSubscribe>(onSubscribe), new Throwable());
```

Comment:
It seems like we'd want to keep a reference to it until we're done with it.  If it goes away, how do we keep track of it?